env/env.obf.main

```diff
@@ -0,0 +1,3 @@
+# This is used by ports.conf in apache2
+APACHE_ARGUMENTS=-D APACHE2_PORT=8002
```

Comment:
While we are changing those files, could we use the same port numbers for all flavors? We don't run them in the same server anymore, so we could just use 8001 (and 8002 if there's a fast track Apache) for all flavors.